scripts/Install-WindowsSDK.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+function Write-Log ([string] $message, $messageColor = "Green") {
+    $currentColor = $Host.UI.RawUI.ForegroundColor
```

Comment:
Why is this not using Write-Host? Also RawUI can be null.

Reply:
Grabbed that from our build script, no idea. Simplifying..